src/components/network/overloadedLinesView.js

```diff
@@ -0,0 +1,178 @@
+                    (intensities[i] / limits[i]) * 100 > fields.overload
+                ) {
+                    fields = {
+                        overload: ((intensities[i] / limits[i]) * 100).toFixed(
+                            1
+                        ),
+                        name: line.name || line.id,
+                        intensity: intensities[i],
+                        load: loads[i],
+                        limit: limits[i],
+                        // conversion [r,g,b] => #XXXXXX ; concat '0' to (color value) in hexadecimal keep last 2 characters
+                        //eslint-disable-next-line
+                        color: '#' + color.map((c) => ('0' + Math.max(c, 0).toString(16)).slice(-2)).join(''),
+                    };
+                }
+            }
+            return fields;
+        };
+
+        setLines(
+            props.lines
+                .map((line) => makeData(line))
+                .filter((l) => l.overload > props.lineFlowAlertThreshold)
+        );
+    }, [props.lines, props.network, props.lineFlowAlertThreshold]);
+
+    function MakeCell(label, color) {
+        return (
+            <TableCell
+                component="div"
+                className={clsx(classes.tableCell, classes.flexContainer)}
+                variant="body"
+                style={{
+                    height: rowHeight,
+                    color: color,
+                }}
+                align={'right'}
+            >
+                {label}
+            </TableCell>
+        );
+    }
+
+    function renderOverloadedLines() {
+        return (
+            <VirtualizedTable
+                className={classes.table}
+                rowCount={lines.length}
+                rowGetter={({ index }) => lines[index]}
+                rowStyle={{ alignItems: 'stretch' }}
+                rowHeight={rowHeight}
+                classes={{ tableRow: classes.rowCell }}
+                columns={[
+                    {
+                        width: 150,
+                        label: intl.formatMessage({ id: 'Name' }),
+                        dataKey: 'name',
+                        cellRenderer: (cellData) =>
+                            MakeCell(
+                                cellData.rowData.name,
+                                cellData.rowData.color
+                            ),
+                    },
+                    {
+                        label: intl.formatMessage({ id: 'Load' }),
+                        dataKey: 'load',
+                        numeric: true,
+                        width: 70,
+                    },
+                    {
+                        label: intl.formatMessage({ id: 'Intensity' }),
+                        dataKey: 'intensity',
+                        numeric: true,
+                        width: 70,
+                    },
+                    {
+                        label: intl.formatMessage({ id: 'Limit' }),
```

Comment:
There is no translation for Limit with an upper case

Reply:
translation added